### ISSUE TITLE
 uninitialized constant Refinery::Authentication (NameError)

### ISSUE THREAD

**Issue by vinchi777** [refinery/refinerycms-blog#452]
when I run 
` rails generate refinery:blog`

I get this 
`.rvm/gems/ruby-2.1.2/gems/activesupport-4.2.3/lib/active_support/inflector/methods.rb:263:in const_get': uninitialized constant Refinery::Authentication (NameError)`

I am in Rails version 4.2.3

**Comment by stats** [comment]
I get this error on rails 4.2.4

To reproduce.

Run the command:
    
    rails new RefineryTest
    cd RefineryTest

add refinerycms gem to the Gemfile

    gem 'refinerycms'

Then run the command

     rails g refinery:cms --fresh-install

Add the refinerycms-blog to the gemfile  

    gem 'refinerycms-blog', git: 'https://github.com/refinery/refinerycms-blog', branch: 'master'

Then run:

    rails g refinery:blog

A short clip of the stack trace

```
    /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/inflector/methods.rb:263:in `const_get': uninitialized constant Refinery::Authentication (NameError)
        from /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/inflector/methods.rb:263:in `block in constantize'
        from /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/inflector/methods.rb:259:in `each'
        from /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/inflector/methods.rb:259:in `inject'
        from /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/inflector/methods.rb:259:in `constantize'
        from /home/nitrous/code/RefineryTest/.bundle/gems/activesupport-4.2.4/lib/active_support/core_ext/string/inflections.rb:66:in `constantize'
        from /home/nitrous/code/RefineryTest/.bundle/bundler/gems/refinerycms-blog-5ee833618c3f/lib/refinery/blog/configuration.rb:33:in `rescue in user_class'
        from /home/nitrous/code/RefineryTest/.bundle/bundler/gems/refinerycms-blog-5ee833618c3f/lib/refinery/blog/configuration.rb:30:in `user_class'
        from (erb):15:in `template'
```

**Comment by stats** [comment]
The solution is to add the **refinerycms-authentication-devise** gem to your gemfile

    gem 'refinerycms-authentication-devise', '~> 1.0.2'

While this allows the generator to run, this might not be a true solution. When trying to login to refinery an error of **Could not find table 'refinery_authentication_devise_roles** occurs.

**Comment by bricesanchez** [comment]
It will be fixed when #449 will be ready and merged. Thanks for reporting.
